db/migrate/20151117225221_add_contest_group_id_to_contest.rb

```diff
@@ -0,0 +1,5 @@
+class AddContestGroupIdToContest < ActiveRecord::Migration
+  def change
+    add_column :contests, :contest_group_id, :integer
```

Comment:
Please add a foreign_key here. See http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_foreign_key

This will ensure that a contest group cannot be deleted, while there are still contests assigned to it.